### PR TITLE
Update 02_Table of Means NSS 70.do

### PR DESCRIPTION
**Pull request by nk-wb-dc**
See some changes to changing negative debentures, and moving asset code after debenture change

### CODE REVIEW

File: nss/Survey_NSS/02_Table of Means NSS 70.do [hunk 1]
```diff
@@ -29,17 +29,20 @@ global r_output "${root}\Data Output Files"
...
-egen durable_other = rowtotal(b7_q5 b8_q5 b9_q4 b10_q3)
-egen real_estate = rowtotal(building_resid b5_1_6 b5_2_6)
 
 *shares & debentures owned by the household in co operative societies & companies as on 30.06.2012 (financial asset should be positive?)
-sum asset b5_1_6 b5_2_6 building_all b7_q5 b8_q5 b9_q4 b10_q3 b11_q6 b12_q3 b13_q4
-replace b11_q6 = 0 if b11_q6<0 //? there's negative value for shares & debentures owned. 
+sum asset b5_1_6 b5_2_6 building_all b7_q5 b8_q5 b9_q4 b10_q3 b11_q6 b12_q3 b13_q4 gold building_resid
 
-gen wealth = asset - debt
+replace b5_2_6 = . if b5_2_6<0 //? there's negative value for shares & debentures owned.
```
Comment (wengxyu1030) [inline nss/Survey_NSS/02_Table of Means NSS 70.do:36]: Can you clarify here? b5_2_6 is land owned by urban households.

Comment (nk-wb-dc) [reply nss/Survey_NSS/02_Table of Means NSS 70.do:36]: > Can you clarify here? b5_2_6 is land owned by urban households.

Thanks. I was just cleaning up assets with negative values. We also need to clean up b11_q6:
image

Comment (nk-wb-dc) [reply nss/Survey_NSS/02_Table of Means NSS 70.do:36]: re: >  
 *shares & debentures owned by the household in co operative societies & companies as on 30.06.2012 (financial asset should be positive?)
-sum asset b5_1_6 b5_2_6 building_all b7_q5 b8_q5 b9_q4 b10_q3 b11_q6 b12_q3 b13_q4
-replace b11_q6 = 0 if b11_q6<0 //? there's negative value for shares & debentures owned. 
+sum asset b5_1_6 b5_2_6 building_all b7_q5 b8_q5 b9_q4 b10_q3 b11_q6 b12_q3 b13_q4 gold building_resid
**Why should building_resid was included in the asset as it's already calculated in the building_all?**

the code is just summing building_resid, it's not included in the asset calculation line 39

Comment (wengxyu1030) [reply nss/Survey_NSS/02_Table of Means NSS 70.do:36]: Thanks for the clarification.

**Pull request merged**